### DISCUSSION TITLE
Meeting Agenda 4/4

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#366] [Q&A]
1. Discuss chat bot for expo booth
3. Review tasks
    1. split up legend task into (a) color assignment abstraction and (b) the actual legend visual component?
5. Discuss changes to visualization
    1. Adding new TopologyView
    2. Removing GlobalView
    3. Reworking RegionView
9. Meet with Darian
   1. turn in weekly report -- canvas assignment has no submission button again

**Comment by moreno-michael** [comment]
Meeting minutes 4/4

1. Chatbot discussion
- Could lead to potential confusion in booth with the stark differences in our product and the premise of an AI chatbot
- Chatbot will be developed as tool for team as internal information bot to be trained on project documentation

2. Task review
- All issues still in progress
- Split legend task into two smaller tasks due to multiple aspects of task

3. Visualization updates
- Add TopologyView
- Remove GlobalView
- Rework RegionView to better illustrate balancer connections and hierarchy that TreeDisplay might not clearly portray

4. Meet with Darian
- Task Review
- Agreed with viability of TopologyView to portray actual connections that are to be routed through the balancer
